src/opdracht2.js

```diff
@@ -0,0 +1,127 @@
+        errorMessageBox.removeChild(previousErrorMessage);
+    }
+
+    if (previousSearchResult) {
+        countryInfoBox.removeChild(previousSearchResult);
+    }
+
+    const URI = `https://restcountries.com`
+    const ENDPOINT = `/v2/name/${name}`
+
+    try {
+        const result = await axios.get(URI + ENDPOINT);
+        const country = result.data[0];
+        console.log(country);
+        //     3. Probeer eens om de _naam_ en _populatie_ van het land te loggen in de console. Welk pad moet je hiervoor volgen?
+        // const response = await axios.get(`https://restcountries.com/v2/name/nederland`);
+
+
+        const countryContainer = document.createElement('article');
+        countryContainer.setAttribute('class', 'search-result-box');
+        countryContainer.setAttribute('id', 'search-result-identifier');
+
+
+        // maak de <span> tag om de titel en vlag in te stoppen
+        const flagTitleContainer = document.createElement('span');
+        flagTitleContainer.setAttribute('class', 'flag-title-container');
+
+        // maak de <img> tag om de vlag in weer te geven
+        const flag = document.createElement('img');
+        // stop de image url in het src attribuut van img
+        flag.setAttribute('src', country.flag);
+        flag.setAttribute('class', 'flag');
+        flagTitleContainer.appendChild(flag);
+
+
+        const countryName = document.createElement('h2');
+        countryName.textContent = country.name;
+        flagTitleContainer.appendChild(countryName);
+
+
+        countryContainer.appendChild(flagTitleContainer);
+
+        //  informatie
+        const population = document.createElement('p');
+        population.textContent = `${country.name} is situated in ${country.subregion}. It has a population of ${country.population} people.`;
+        countryContainer.appendChild(population);
+
+        // // informatie volgende regel
+        const capital = document.createElement('p');
+        capital.textContent = `The capital is ${country.capital} and you can pay with ${createCurrencyDescription(country.currencies)}`;
+        countryContainer.appendChild(capital);
+
+        // voeg de country <div> toe aan de countryContainer
+        countryInfoBox.appendChild(countryContainer);
+
+
+    } catch (err) {
+        console.error(err);
+        const errorMessage = document.createElement('p');
+        // Check welke error message van toepassing is
+        errorMessage.setAttribute('class', 'error-message');
+        errorMessage.setAttribute('id', 'error-message-identifier');
+        errorMessage.textContent = `${name} bestaat niet. Probeer het nogmaals.`;
+
+        errorMessageBox.appendChild(errorMessage)
+    }
+}
+
+
+function createCurrencyDescription(currencies) {
+    let output = 'and you can pay with ';
+
+    if (currencies.length === 2) {
+        return output + `${currencies[0].name} and ${currencies[1].name}'s`;
+    }
+
+    return output + `${currencies[0].name}'s`;
```

Comment:
Dit is niet echt uitbreidbaar voor als er meer dan twee valuta's zouden zijn. Wellicht dat je de volgende keer iets met een for-loop of de [Array.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) functie kan doen